example.yaml

```diff
@@ -19,6 +19,9 @@ aggregates:
  - "min"
  - "max"
  - "count"
+influx_address: http://localhost:8086
```

Comment:
Can we move these to a separate section, just to keep things visually organized?